docs: added README to all packages (#453)
KILTprotocol/sdk-js: packages/types/README.md

[![](https://user-images.githubusercontent.com/39338561/122415864-8d6a7c00-cf88-11eb-846f-a98a936f88da.png)
](https://kilt.io)

![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)

# KILT Types

> This package provides the generally applicable TypeScript types for different mechanisms in the KILT SDK.

Reviewer: ```suggestion
This package provides the generally applicable TypeScript types for different operations provided by the KILT SDK.
```

Author: Hmm, not sure if operations is enough here. I wanted to point out, that it contains all the base types you would also need to rebuild an SDK basically (while currently not knowing, if that is true)

Author: ah, but I say this in the next paragraph... so maybe it is fine to take your suggestion

Author: a05286eff11b655e0641df1ce8938f24b4bad710

Author: 5fed91be8f69ca7a08bf7cad96755c93aec05f74

> This is also contains all the types, which someone would need to support for working with KILT credentials and DIDs not using the SDK.

Reviewer: ```suggestion
The package also contains all the types, which someone would need to support for working with KILT credentials and DIDs not using the SDK.
```

Author: a05286eff11b655e0641df1ce8938f24b4bad710
